pom.xml

```
@@ -496,6 +496,10 @@ and
           </systemPropertyVariables>
           <!-- SUREFIRE-1588 workaround; too late for systemProperties: -->
           <argLine>-Djdk.net.URLClassPath.disableClassPathURLCheck=true</argLine>
+          <environmentVariables>
+            <!-- Use local version of the plugin over the released one ATH would otherwise use -->
+            <LOCAL_JARS>target/job-config-history-plugin.hpi</LOCAL_JARS>
```

The artifact ID is `jobConfigHistory`, so this should be

```suggestion
            <LOCAL_JARS>target/jobConfigHistory.hpi</LOCAL_JARS>
```